compiler/one-cmds/one-quantize

```diff
@@ -218,9 +219,29 @@ def _set_default_values(args):
             args, 'input_dtype'):
         setattr(args, 'input_model_dtype', 'float32')
     if not _utils._is_valid_attr(args, 'quantized_dtype'):
-        setattr(args, 'quantized_dtype', 'uint8')
+        if _utils._is_valid_attr(args, 'quant_config'):
+            # Get quantized_dtype from qconfig file
+            try:
+                with open(getattr(args, 'quant_config')) as f:
+                    qconf = json.load(f)
+                    if 'default_quantization_dtype' in qconf:
+                        setattr(args, 'quantized_dtype', qconf['default_quantization_dtype'])
+            except json.decoder.JSONDecodeError:
+                print('Failed to decode ' + getattr(args, 'quant_config') + '. Please check it is a json file.')
```

Comment:
This exception is tested by one-quantize_neg_020.test

Reply:
what if json file is valid but `default_quantization_dtype` doesn't existing in json ?

Reply:
Then, it will use default quantization_dtype, i.e., uint8, as before.

Reply:
But `setattr(args, 'quantized_dtype', 'uint8')` is not called... or is there somewhere else?

Reply:
It is called if both quantized_dtype and quant_config are not given. The code is right below the comment (line 232) :)

Reply:
I think if something like this case
```json
{
    "default_quantization_dtype" : "uint8",
    "layers" : [
        {
            "names" : ["InceptionV3..., ..."],
            "dtype" : "int16",
            "granularity" : "channel"
        }
    ]
}
```
or
```json
{
    "default_quantization_dtype" : "uint8",
    "default_granularities" : "channel",
    "layers" : [
        {
            "names" : ["InceptionV3..., ..."],
            "dtype" : "int16",
            "granularity" : "channel"
        }
    ]
}
```

Reply:
Ah.. Sorry, I misunderstood the comment. I've updated the PR. Thanks!